webpack/assets/javascripts/react_app/components/SettingsTable/SettingsTableHelpers.js

```diff
@@ -10,17 +9,13 @@ export const withTooltip = Component => componentProps => {
   const { tooltipId, tooltipText, ...rest } = componentProps;
 
   return (
-    <OverlayTrigger
-      overlay={<Tooltip id={tooltipId}>{tooltipText}</Tooltip>}
-      trigger={['hover', 'focus']}
-      placement="top"
-      rootClose={false}
-    >
-      {/* The span is needed because OverlayTrigger overrides child events */}
+    <Tooltip content={tooltipText}>
+      {/* The span is needed because Tooltip overrides child events */}
+      {/* TODO check  */}
```

Comment:
leftover?

Reply:
yup, checked the comment and the `span` is still needed